src/SourceBuild/content/build.sh

```diff
@@ -33,12 +34,14 @@ usage()
   echo ""
 
   echo "Advanced settings:"
-  echo "  --build-tests                   Build repository tests. May not be supported with --source-only"
+  echo "  --build-repo-tests              Build repository tests. May not be supported with --source-only"
   echo "  --ci                            Set when running on CI server"
   echo "  --clean-while-building          Cleans each repo after building (reduces disk space usage, short: -cwb)"
   echo "  --excludeCIBinarylog            Don't output binary log (short: -nobl)"
   echo "  --prepareMachine                Prepare machine for CI run, clean up processes after build"
   echo "  --use-mono-runtime              Output uses the mono runtime"
+  echo "  --testnobuild                   Run tests without building when invoked with --test"
+  echo "  --test-projects <NAMES>         Run tests for the specified projects"
```

Comment:
The name and description can give the impression that the `--test` option is not necessary.

Reply:
I've reverted the changes to support this: https://github.com/dotnet/installer/commit/28ddf277572e32ae696a73f2dd4c2d38b714040f